test/components/CalendarDay_spec.jsx

```diff
@@ -44,6 +44,10 @@ describe('CalendarDay', () => {
       handleDayClickSpy = sinon.spy(CalendarDay.prototype, 'handleDayClick');
     });
 
+    afterEach(() => {
```

Comment:
Does [this file](https://github.com/airbnb/react-dates/blob/master/test/_helpers/restoreSinonStubs.js) not run in karma?